### DISCUSSION TITLE
Unable to view raw file

### DISCUSSION THREAD

**Discussion by sabharwal2001** [community/community#41519] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am unable to "view raw" files of my repository. I checked it from different locations. It is opening fine in Belgium, Poland and Russia but gives "request timeout" error when I try to open it from India. https://raw.githubusercontent.com/sabharwal2001/news/main/home.json

**Comment by airtower-luna** [comment]
Some people mentioned similar issues with a specific provider in India: https://github.com/orgs/community/discussions/32889#discussioncomment-4320144

If you're with the same provider, you might want to contact their customer support and ask them to fix it.

**Comment by vipulbehl** [comment]
This is an issue with the DNS, mostly happening in India with people using Jio Fibre connection. Adding 8.8.8.8 in your DNS will resolve the issue.

Adding DNS on [UBUNTU](https://www.how2shout.com/linux/how-to-set-dns-nameserver-on-ubuntu-22-04-lts-jammy/) [WINDOWS](https://pureinfotech.com/change-dns-server-address-windows-11/)

**Reply by khushal3012** [reply]
this works, thank you

**Reply by SahilMepani** [reply]
This is not working anymore

**Reply by vipulbehl** [reply]
@SahilMepani Can you provide more details, what OS, browser and ISP provider are you using? I checked on my system atleast it is working.

**Reply by SahilMepani** [reply]
It was working for me till yesterday evening. I already have my DNS changed to google DNS.

But from the evening it stopped working. I thought maybe something changed on my Desktop but I tried my Macbook too and it stopped working there as well. 

I'm on Windows, JIO Fibre and doesn't work on any browser. Not even by using Opera VPN

**Reply by SahilMepani** [reply]
I also have "185.199.108.133 raw.githubusercontent.com" in my host file

**Reply by SahilMepani** [reply]
I just used proton VPN and it is working. So basically I have to enable the VPN when I need to connect raw.githubusercontent.com

**Reply by Suriya-K7** [reply]
![Screenshot (43) 0](https://github.com/community/community/assets/121870740/b6c5c5db-ed8a-43ce-ae9b-58e75ba70f3a)
Thanks  bro it's now working

**Reply by Shanthi2814** [reply]
Thank you now it's working!

**Reply by biswa-mohapatra** [reply]
Thank you for the solution. The issue got resolved!

**Comment by anvesh9652** [comment]
Still, if anyone is facing the same issue, follow the page below:

https://developers.google.com/speed/public-dns/docs/using
